### PR TITLE
Update assets-and-images.md

### PR DESCRIPTION
**Pull request by a14n**
Remove `material-design-icons` mentions

### CODE REVIEW

File: assets-and-images.md [hunk 1]
```diff
@@ -18,12 +18,8 @@ file might include something like the following:
...
       - assets/background.png
-    material-design-icons:
-      - name: action/home
-      - name: content/add
 
-Both of these sections - the `assets` section and the `material-design-icons`
-section - specify files that should be included with the application. Flutter
+The `assets` section specify files that should be included with the application. Flutter
 places them in a special archive called the _asset bundle_, which applications
 can read from at runtime.
```
Comment (Hixie) [inline assets-and-images.md]: the next paragraph needs updating too.

**Comment by a14n** [comment]
ptal

**Comment by Hixie** [comment]
lgtm

**Comment by krisgiesing** [comment]
LGTM

**Comment by sethladd** [comment]
thanks!

**Pull request merged**